site/2015/04/02/orm-2-5-0.rst

```diff
@@ -0,0 +1,1461 @@
+Doctrine ORM 2.5.0 Release
+==========================
+
+We are happy to announce the immediate availability of Doctrine ORM
+``2.5.0``.
+
+This release spans over almost 2 years of development, and is a major
+effort by the team and the community to make the ORM more robust and
+performing.
+
+`457
+issues <http://www.doctrine-project.org/jira/browse/DDC-3322?jql=project%20%3D%20DDC%20AND%20fixVersion%20%3D%202.5%20ORDER%20BY%20status%20DESC%2C%20priority%20DESC>`__
+were solved in this release, so we are very proud of the work being done
+by the community and the core team.
+
+What is new in 2.5.x?
+---------------------
+
+Doctrine ORM 2.5.0 comes with a set of major improvements:
+
+-  The Second Level Cache, a component that greatly improves ORM
+   performance
+-  Embeddable classes, allowing for a more fine-grained design of your
+   entities without having to resort to one-to-one associations for
+   Value Objects
+-  Entity type specific event listeners, for improved event handling
+   performance
+-  Improvements in the Criteria Collection filtering API, now also
+   supporting ``EXTRA_LAZY`` filtering
+
+What has to be done to upgrade to 2.5.x?
+----------------------------------------
+
+Some Backwards incompatible changes were also involved in this release:
+to read them, along with a more extensive list of the 2.5.0 changes,
+please consult `the upgrade
+notes <http://docs.doctrine-project.org/en/latest/changelog/migration_2_5.html>`__.
+
+Stability
+---------
+
+We currently do not have a release schedule for Doctrine ORM ``2.6.0``.
+
+As of today, Doctrine ORM ``2.5.x`` is our stable distribution, and will
+receive regular bugfix releases.
+
+Doctrine ORM ``2.4.8`` will be the last bugfix release for the ``2.4.x``
+series. Further releases will only occur in the eventuality of a
+security issue being discovered.
+
+We will also keep patching previous versions of the ORM in the
+eventuality of a security issue being discovered.
+
+Installation
+------------
+
+You can install this version of the ORM by using Composer and the
+following ``composer.json`` contents:
+
+.. code:: json
+
+
+    {
+        "require": {
+            "doctrine/orm": "2.5.0"
```

Comment:
Would it be wise to suggest `~2.5.0` here? Given that BC breaks might happen between your minor versions (if not in API, then just in functionality, as you mentioned above), you probably don't want to suggest `~2.5` or `^2.5`.